src/array/tests/reduceRight.test.js

```diff
@@ -0,0 +1,12 @@
+import reduceRight from '../reduceRight';
+
+describe('array/reduceRight', () => {
+  it('uses the built in array reduceRight method', () => {
+      const input = [10, 12, 14, 15];
```

Comment:
Indentation is off on this line. I guess that’s why the CI is failing on this PR. :)